docs/admin/configuration.md

```diff
@@ -48,12 +50,14 @@ proceed_on = ["pass", "dispatcher_return"]
 
 ## bpfctl
 
-bpfctl expects a configuration file to be present at `/etc/bpfd.toml`.
+bpfctl expects a configuration file to be present at `/etc/bpfctl/bpfctl.toml`.
 If no file is found, defaults are assumed.
+The `scripts/setup.sh` commands will copy a default version of the file to the correct location
+from `scripts/bpfctl.toml`, which can then be overwritten if needed.
 
 ```toml
 [tls] # REQUIRED
   ca_cert = "/etc/bpfd/certs/ca/ca.pem"
-  cert = "/etc/bpfd/certs/bpfctl/bpfctl.pem"
-  key = "/etc/bpfd/certs/bpfctl/bpfctl.key"
+  cert = "/etc/bpfctl/certs/bpfctl/bpfctl.pem"
```

Comment:
nit: can we get rid of the nested bpfctl dir now and just have `/etc/bpfctl/certs/bpfctl.pem`

Reply:
I currently have `/etc/bpfctl/certs/gocounter/` as well.